documentation/airport-guides/klax.md

```diff
@@ -0,0 +1,113 @@
+
+NOTE: As already mentioned above, special attention must be paid to aircraft on the GOATZ1 and DIRBY1 arrivals, which enter the airspace to the South. To prevent aircraft from exiting our airspace, a shortcut can be given to waypoints ZAPPP/BUFIE/SLI/TRNDO, as aircraft first enter the airspace near Catalina Island.
+
+#### Operation
+
+##### Preferential RWYs
+- With prevailing winds from the West, departures will be off of 24L and 25R, and arrivals will be to 24R and 25L with arrivals from the South and East usually landing 25L and from the North and West landing on 24R
+- KLAX usually operates an "Over Ocean" configuration at night, where aircraft will depart from runways 24L and 25R and land in the opposite direction on 6L and 7R. This configuration is not simulated in openScope.
+
+### Airports of Interest
+
+#### KLAX (Primary Airport)  
+Los Angeles International Airport  
+710,989 annual operations  
+
+The primary airport of the Kennedy area of the New York TRACON, New York's John F Kennedy Int'l Airport is the second busiest airport in N90, behind Newark Liberty Int'l (KEWR). It is the 19th busiest in the world, and 10th busiest in the United States.
+
+#### KHHR
+Jack Northrop Field/Hawthorne Municipal Airport  
+80,793 annual operations  
+3.8nm East of KLAX  
+
+This small GA airport is in very close proximity to the considerably larger Los Angeles International Airport so special consideration must be paid to avoid encroaching on the LAX airspace. The airport is the home base for approximately 150 general aviation aircraft, and multiple flight training schools. Tesla Design Studio, SpaceX and The Boring Company are all located at the airport.
+
+#### KTOA
+Torrance/Zamperini Field Airport  
+135,620 annual operations  
+9.0nm Southeast of KLAX  
+
+Zamperini Field is a small airport three miles (5 km) southwest of downtown Torrance, in Los Angeles County. It's home to the Western Museum of Flight as well as the Robinson Helicopter Company. Their entire production, assembly, and testing facilities are on the southeast side of the airfield and are the largest buildings at the field.
+
+### Additional Resources
+<a href="https://www.youtube.com/watch?v=UHwg3yrOgtE" target="_blank">VASAviation: Los Angeles Stadium Sector (West Configuration)</a>
+<a href="https://laartcc.org/uploads/lightbox/KLAX%20FINALS%20WEST.png" target="_blank">VATSIM vZLA's Vectoring Diagram</a>
+<a href="https://laartcc.org/document/socal-tracon-general-sop" target="_blank">VATSIM vZLA's SCT General SOP</a>
+<a href="https://laartcc.org/document/socal-tracon-area-2-los-angeles" target="_blank">VATSIM vZLA's SCT Los Angeles Area SOP</a>
+<a href="https://laartcc.org/document/socal-tracon-area-6-del-rey" target="_blank">VATSIM vZLA's SCT Del Rey Area SOP</a>
```

Comment:
I'd suggest adding two whitespaces at the end of each line in order to have a single line for each line.